dcompiler.dd

```diff
@@ -717,7 +714,7 @@ $(COMMENT
 		$(D unittest) $(LINK2 version.html#PredefinedVersions, version identifier)
 	  )
 	  $(SWITCH $(SWNAME -v),
-		verbose
+		enable verbose output for each compiler process
```

Comment:
"each compiler process" = parse, import, semantic, semantic2, semantic3, codegen, ... They are what you can see when you use `-v` for a compilation.
And of course, -v does not work to see generated AST, IR, assembly, etc. I think the single word "verbose" is too rough explanation. That's why I updated the text here a little.